src/components/Card/src/Card/card.css

```diff
@@ -0,0 +1,180 @@
+:root {
+    --denhaag-card-arrow-left: 0;
+    --denhaag-card-arrow-position: relative;
+    --denhaag-card-arrow-right: 28px;
+}
+
+.denhaag-card {
+  width: var(--denhaag-card-width);
+  border-radius: var(--denhaag-card-border-radius);
+}
+
+.denhaag-card__wrapper {
+    border-radius: var(--denhaag-card-border-radius);
+    border-width: var(--denhaag-card-border-width);
+    border-color: var(--denhaag-card-border-color);
+    border-style: solid;
+    box-sizing: border-box;
+    height: var(--denhaag-card-height);
+    padding-inline: var(--denhaag-card-padding-inline);
+    padding-block: var(--denhaag-card-padding-block);
+    flex-direction: column;
+    display: flex;
+}
+
+.denhaag-card:focus .denhaag-card--case__text__wrapper {
+  outline: var(--denhaag-focus-border);
+  position: relative;
+}
+
+.denhaag-card:focus {
+  outline: none;
+}
+
+.denhaag-card__arrow {
+    left: var(--denhaag-card-arrow-left);
+    position: var(--denhaag-card-arrow-position);
+    right: var(--denhaag-card-arrow-right);
+}
+
+.denhaag-card__title {
+    color: var(--denhaag-black);
```

Comment:
We shouldn't be referencing to proprietary styles in the component CSS. It's probably a good idea to add a token for the title font color.